doc/destructors.md

```diff
@@ -731,7 +731,7 @@ used to specialize the object traversal in order to avoid deep recursions:
       if x.left != nil: s.add(x.left)
       if x.right != nil: s.add(x.right)
       # free the memory explicit:
-      dispose(x)
+      `=dispose`(x)
```

Comment:
The =dispose hook is not officially supported. It should be `=destroy(x[])` or something like that.

Reply:
```nim
template `=dispose`*[T](x: owned(ref T) =
  ...
  let hdrSize = align(sizeof(RefHeader), alignment)
  alignedDealloc(p -! hdrSize, alignment)
```
Isn't `=dispose(x)` used to free the pointer `x`. Is it equivalent to `=destroy(x[])`?

Reply:
Ok.